build/test.sh

```diff
@@ -60,7 +60,7 @@ check_dependencies() {
 check_dependencies
 
 echo "Running tests:"
-go test -v ${TARGETS} -list .
+# go test -v ${TARGETS} -list .
```

Comment:
Do we want to remove the line?

Reply:
This line slows down the tests, but it  doesn't have to be a part of this PR.